beamer/health/check.py

```diff
@@ -389,6 +441,26 @@ def analyze_transfers(transfers: TransferMap, ctx: Context) -> None:
         else "Everything is calm in Beamerland."
     )
 
+    def format_token_volume(token_amount: int, token_details: TokenDetails) -> str:
+        return (
+            str(get_token_amount_in_decimals(token_amount, token_details))
+            + " "
+            + token_details["symbol"]
+        )
+
+    volume_per_token = ", ".join(
+        [
+            format_token_volume(token_amount, ctx.tokens[token_symbol])
+            for token_symbol, token_amount in ctx.stats.volume.items()
+        ]
+    )
+    expired_request_volume_per_token = ", ".join(
+        [
+            format_token_volume(token_amount, ctx.tokens[token_symbol])
+            for token_symbol, token_amount in ctx.stats.expired_requests_volume.items()
+        ]
+    )
```

Comment:
Similar to above, I would use a generator expression here instead of a list comprehension.